libcontainer/container_linux.go

```diff
@@ -779,3 +780,28 @@ func (c *linuxContainer) currentState() (*State, error) {
 	}
 	return state, nil
 }
+
+func (c *linuxContainer) Kill(sigStr string) error {
```

Comment:
I don't think that we need to parse a signal name in libcontainer. I would prefer to use os.Signal here.